dev/local_flask_lineage_server.py

```diff
@@ -0,0 +1,33 @@
+import json
+import logging
+import os
+
+from dateutil.parser import parse
+from flask import Flask, request
+
+APP = Flask(__name__)
```

Comment:
can we change the APP to smaller case app?

Reply:
Why so? It is a module level constant and should it not be in Upper case?